pkg/tsdb/azuremonitor/types/types.go

```diff
@@ -139,15 +140,27 @@ type AzureMonitorJSONQuery struct {
 // AzureMonitorDimensionFilter is the model for the frontend sent for azureMonitor metric
 // queries like "BlobType", "eq", "*"
 type AzureMonitorDimensionFilter struct {
-	Dimension string `json:"dimension"`
-	Operator  string `json:"operator"`
-	Filter    string `json:"filter"`
+	Dimension string   `json:"dimension"`
+	Operator  string   `json:"operator"`
+	Filter    []string `json:"filter"`
```

Comment:
From our conversation: suggested moving this change to a separate PR

Reply:
Yeah I'm not sure I follow. Why is this needed? Unless migrated properly in frontend and backend, it will break any existing queries passing `Filter` as a string. Would be good with a new issue that provides context on why this change is necessary.